.github/workflows/scheduled_tests.yml

```
@@ -0,0 +1,51 @@
+name: Scheduled Tests
+
+on:
+  schedule:
+    - cron: "0 22 * * *" # 7pm Brazil, 10pm UTC, 8am AEST
+
+jobs:
+  python-tests-all-versions:
+    strategy:
+      max-parallel: 9
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

```suggestion
        os: [ubuntu-latest, macos-latest]
```
I think we could also remove windows runner here.

---

I am not sure about that. If we remove it from here, SyMPC will never get tested on windows.

---

I think we should also test the windows node here - since those jobs should not "bother" the devs. Could we put those at an hour like 2:00 AM UTC?

---

The reason it was removed was that ,it was very slow on the CI tests and failed frequently,so it was removed.
Maybe we could test it,when we are releasing a package or stable release @gmuraru

---

Yeah at schduled_tests we could have it